dp/cloud/helm/dp/README.md

```diff
@@ -0,0 +1,186 @@
+
+Domain-proxy
+===========
+
+A Helm chart for magma orchestrator's domain-proxy module.
+
+This Chart will deploy the following:
+
+- 1 x Configuration controller
+- 1 x protocol controller with 80/443 TCP port exposed with either ingress/contour ingress resource.
+- 1 x radio controller with GRPC 50053/TCP port.
+- All using Kubernetes Deployment
+- 1 x job for Postgres database migration.
```

Comment:
Can you point me to the migration / what specifically is occurring -- is it just initializing the relevant tables?

Reply:
Sure, 
we have specific service which is responsible for running database migrations. You can find it here:
https://github.com/magma/domain-proxy/tree/master/db_service
Basically it's just initialize tables required by DP.